R/backend-tuneR_backend.R

```diff
@@ -21,11 +21,18 @@ tuneR_read_mp3_or_wav <- function(filepath, from = 0, to = Inf, unit = "samples"
       to_ <- max(to_, from_ + 0.015)
       to_ <- 0.05 + to_*1.01
       # mac throws permission denied when not set to afplay
-      afplay_path <- system("which afplay", intern = TRUE)
-      if(grepl("^darwin", R.version$os) & grepl(afplay_path, tuneR::getWavPlayer())) {
+      afplay_path <- Sys.which("afplay")["afplay"]
+      if(nzchar(afplay_path) && grepl("^darwin", R.version$os) && grepl(afplay_path, tuneR::getWavPlayer())) {
         tuneR::setWavPlayer(afplay_path)
       }
-      wave_obj <- monitoR::readMP3(filepath, from = from_, to = to_)
+
+      # monitoR::readMP3 needs libmp3splt to be installed, which will not normally
+      # be the case
+      if (nzchar(Sys.which("mp3splt")["mp3splt"])) {
+        wave_obj <- monitoR::readMP3(filepath, from = from_, to = to_)
```

Comment:
Do we need monitoR at all? I think we should just go with tuneR for everything. This might have been committed by accident, maybe because monitoR is faster?

Reply:
Yeah exactly! I just left it in for, well, because I don't know about further plans, but the function is not used anywhere! Then I'll gladly remove it.

BTW, were you thinking that `tuneR` should be replaced, at some time, by the new `wav` package?